addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1733,6 +1733,8 @@ const RangeUserValueWidget = UnitUserValueWidget.extend({
         this.input.setAttribute('max', max);
         this.input.setAttribute('step', step);
         this.containerEl.appendChild(this.input);
+
+        this._onInputChange = _.debounce(this._onInputChange, 500);
```

Comment:
Is there a reason for why you chose 500 ? If not, I think it would be better to choose a smaller number (like 100) because I find it slow to display the changes with 500. For example, if you click on the lowest quality in the image quality option, you clearly wait before seeing the pixelated image. The same goes for when you press quickly the arrows, you have to wait a bit when you stop (and so you have to slow down to see the result of each pressing).